sym_api_client_python/services/abstract_datafeed_event_service.py

```diff
@@ -158,12 +146,15 @@ def remove_suppression_listener(self, suppression_listener):
     # TODO: Add doc
     # TODO: Change all brackets access types by get function
     def handle_events(self, events):
-        """ Routes the event to the proper handler. Also Checks to see that the sender's email is not equal to Bot's email in config file.
+        """ Routes the event to the proper handler.
 
+            Also Checks to see that the sender's email is not equal to Bot's email in config file.
             This functionality helps the bot avoid entering an infinite loop where it responds to its own messageSent
             after an event comes back over the dataFeed
         """
         log.debug('DataFeedEventService/handle_events()')
+        log.debug(events)
+        print(events)
```

Comment:
Please delete the print statement as you already use the logger above

Reply:
fixed